api/ServiceWorkerGlobalScope.json

```diff
@@ -1079,7 +1089,7 @@
           "spec_url": "https://notifications.spec.whatwg.org/#dom-serviceworkerglobalscope-onnotificationclick",
           "support": {
             "chrome": {
-              "version_added": "40"
+              "version_added": "42"
```

Comment:
I would suggest making this change separately. A small change in the version number requires a closer look to see if the old data was actually correct and a note is missing. (Also Chrome for Android is left unchanged below.)